src/HealthChecks.System/DependencyInjection/SystemHealthCheckBuilderExtensions.cs

```diff
@@ -218,6 +218,9 @@ public static class SystemHealthCheckBuilderExtensions
     /// <param name="tags">A list of tags that can be used to filter sets of health checks. Optional.</param>
     /// <param name="timeout">An optional <see cref="TimeSpan"/> representing the timeout of the check.</param>
     /// <returns>The specified <paramref name="builder"/>.</returns>
+#if NET6_0_OR_GREATER
+    [global::System.Runtime.Versioning.SupportedOSPlatform("windows")]
```

Comment:
just for my own education: what is the benefit of using `global` prefix here?

Reply:
Normally I wouldn't fully qualify the attribute, but I got a warning that the using wasn't necessary for the netstandard2.0 build. So it would take another `#if` in the usings. So I thought it was easier to just qualify it.

The assembly/namespace is `HealthChecks.System`. So without the `global::` prefix, `System` meant the `HealthChecks.System` namespace, which doesn't have a `Runtime` subnamespace.